docs/web_reference.rst

```diff
@@ -1130,38 +1130,61 @@ duplicated like one using :meth:`Application.copy`.
 
    .. method:: make_handler(**kwargs)
 
-      Creates HTTP protocol factory for handling requests.
-
-      :param kwargs: additional parameters for
-                     :class:`~aiohttp.web.RequestHandlerFactory`
-                     constructor.
-
-        .. deprecated:: 1.0
-          You should not pass ``debug`` parameter within ``kwargs`` since
-          its usage in :meth:`Application.make_handler` is deprecated in favor
-          of :attr:`Application.debug`.
-          The :class:`Application`'s debug mode setting should be used
-          as a single point to setup a debug mode.
-
-      You should pass result of the method as *protocol_factory* to
-      :meth:`~asyncio.AbstractEventLoop.create_server`, e.g.::
-
-         loop = asyncio.get_event_loop()
-
-         app = Application(loop=loop)
-
-         # setup route table
-         # app.router.add_route(...)
-
-         await loop.create_server(app.make_handler(),
-                                  '0.0.0.0', 8080)
-
-      ``secure_proxy_ssl_header`` keyword parameter
-      can be used to detect request scheme::
-
-         await loop.create_server(app.make_handler(
-            secure_proxy_ssl_header='X-Forwarded-Proto'),
-            '0.0.0.0', 8080)
+    Creates HTTP protocol factory for handling requests.
+
+    :param handler: Request handler. Default:
```

Comment:
Let's drop it from doc.
It's a hidden param, not part of public API (at least now now).